src/spikeinterface/extractors/cbin_ibl.py

```diff
@@ -117,6 +121,7 @@ def __init__(self, folder_path, load_sync_channel=False, stream_name="ap"):
         self._kwargs = {
             "folder_path": str(Path(folder_path).absolute()),
             "load_sync_channel": load_sync_channel,
+            "cbin_file": str(Path(cbin_file).absolute()),
```

Comment:
resolve is better than absolute but probably does not matter. Resolve gets symbolic links and makes the path absolute.

Reply:
done